lib/features/common/domain/entity/tokens/tokens_entity.dart

```diff
@@ -0,0 +1,20 @@
+// ignore_for_file: depend_on_referenced_packages
+
+import 'package:freezed_annotation/freezed_annotation.dart';
+
+part 'tokens_entity.freezed.dart';
+
+/// {@template tokens_entity.class}
+/// Tokens entity
+/// {@endtemplate}
+@freezed
```

Comment:
only for this file we have added `freezed` to dependencies? may be better do it without `freezed`?

Reply:
✅